src/Models/Constants/Interfaces.ts

```diff
@@ -312,6 +320,14 @@ export interface IAzureStorageAccountPropertyData {
     primaryEndpoints: { blob: string; [additionalProperty: string]: any };
     [additionalProperty: string]: any;
 }
+
+export interface IAzureTimeSeriesDatabaseConnectionPropertyData {
+    connectionType: 'AzureDataExplorer' | string;
+    adxEndpointUri: string; // cluster url
```

Comment:
nit: can we move the comment to the /** syntax on the line above so it shows on the intellisense?

Reply:
ahh I will learn this one day... :) fixing it